content/guest-essays/uncertainty-and-utilitarianism.md

```diff
@@ -0,0 +1,197 @@
+
+What shall you do when hedging does not apply? It is clear that you will have to make some trade-offs between the moral views you are considering, based on your degrees of confidence in them and the degrees of value they assign to your options. It is also clear  that if your confidence splits evenly across two moral hypotheses, then the hypothesis that sees a greater difference between the options calls the shot. This means that if you are equally certain in the considered hypotheses in the new versions of the restaurant case and the donation case, then the moral hypothesis that sees the greatest difference between the options wins. So, if vegetarianism sees a greater difference between eating vegetarian and eating meat than common-sense does, then you should eat vegetarian. Similarly, if the donation-friendly view sees a greater difference between donating and not donating than common-sense does, then you should donate. Of course, there is no guarantee that these value differences hold between the options, but they seem plausible at least for the donation case, where seemingly much more is at stake for the utilitarian-friendly view than for the common-sense view: saving lives versus bearing a significant but not very severe cost to oneself.
+
+Of course, your confidence does not split evenly in all cases. In most cases, you have some significant confidence in several views, but you have more confidence in some views than in others. Cases where you are more confident in the view that sees the greatest difference between the options pose no difficulty, since here it is obvious that this view rules. For example, if, as a true utilitarian, you are more convinced in the donation-friendly view that also sees the greater value difference between donation and non-donation, then you should donate.
+
+It is more controversial what you should do when the confidence and the value difference do not line up nicely in this way. Here we can follow the lead of what is reasonable to do under empirical uncertainty.[^7]
+
+Suppose Julia considers whether to speed around a blind corner. She thinks it is unlikely that anyone is crossing the road immediately around the corner, but she is not sure. If she speeds and hits someone, she will certainly severely injure them. If she goes slowly, she certainly will not injure anyone, but will get to work slightly later. Here the right option is clearly not to speed, even though it is unlikely that someone will cross the road. The situation has the following structure (the numbers are only illustrative):
+
+|               |  Someone will cross <br> (Probability: 0.2) |No one will cross <br> (Probability: 0.8) |
```

Comment:
@rychappell I wonder if the table will look better splitting the line after `<br>` into a dedicated row:

|    | Someone will cross | No one will cross | 
|  --- | --- | --- |
| Probability | 0.2 | 0.8 |
| **Speed** | -20 | 15 | 
| **Not speed | 10 | 10 |

Reply:
Best to leave as is (i.e. probabilities included in the titles); too confusing to have some rows describing probabilities and some describing utilities.